lxd/device/nic_bridged.go

```diff
@@ -246,13 +254,13 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 			} else {
 				// If MAAS isn't being used, then static IP cannot be used with unmanaged parent.
 				if d.config["ipv4.address"] != "" && d.config["maas.subnet.ipv4"] == "" {
-					return fmt.Errorf("Cannot use manually specified ipv4.address when using unmanaged parent bridge")
+					return fmt.Errorf("Cannot have ipv4.address as none unless using security.ipv4_filtering")
```

Comment:
This bit should stay the same, remember that empty string and "none" are different meanings.

Reply:
We need to allow the "none" value though to not trigger this error though when using IP filtering.